plugins/angr_binsync/controller.py

```diff
@@ -35,10 +35,10 @@ def active_context(self):
         except NotImplementedError:
             return None
 
-        if func is None:
+        if func is None or func.am_obj is None:
             return None
 
-        return func.addr
+        return func.name, func.addr
```

Comment:
@frqmod I don't like this checking for `func.name`. Can we instead make this only the func.addr again?

Reply:
yes